### DISCUSSION TITLE
Add validation for image manipulation names to not be numeric

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#2884] [Feature Requests]
I prefer to name my image manipulations by size. Like, `1600` for 1600px width.

![13e59f1d-f93a-4416-9ad2-f5168076d423](https://user-images.githubusercontent.com/12234507/188845707-f6ff3f91-15f2-4eb3-a508-5dbea1812618.png)

But in some cases (I'm not sure how to reproduce) they just do not work in templates. 
I mean, `{meta_ogimage:1600}` in template renders as `{meta_ogimage:1600}` in html.

I tried to rename image manipulation to `i1600` and it worked, now in html I see link `https://domain.local/images/_i1600/image.jpg` instead of EE tag. 

**But**, there is another problem now: image folders do not get new names after renaming image manipulation name, and that link points to a non-existing folder.

**Comment by intoeetive** [comment]
Using  numbers as modifier (in this case, manipulation name) is not recommended - is there is overlap with entry ID or some other ID, the system might behave in an unexpected way. Always use a non-numeric prefix (I'll classify this issue as feature request to add validation for this)

And correct, there is no way to rename the manipulation and keep the files, - it's a filesystem-based thing, so the folder name will be changed.

You now need to synchronize your upload directory

**Comment by shbchk** [comment]
Okay, thanks, Yuri!

**Comment by intoeetive** [comment]
This will be implemented with merge of https://github.com/ExpressionEngine/ExpressionEngine/pull/3367 (EE6) and https://github.com/ExpressionEngine/ExpressionEngine/pull/3366 (EE7)
